misc/mapdeps.erl

```diff
@@ -92,7 +102,9 @@ map_rebar(BaseDir, Path, Acc) ->
                                            atom_to_list(DepName),
                                            "rebar.config"]),
                               map_rebar(BaseDir, DepPath, NA)
-                      end
+                      end;
+                 ({_DepName, _, {_,_,_}  }, A) ->  
```

Comment:
@binarytemple what's this function case for? Am i being dense? Looks like the above case will always match

Reply:
Well spotted - let me remove that